generator/annual-report/github_issue.tmpl

```diff
@@ -14,11 +14,13 @@ Actions for the chair/organizer of the community group:
 Once all the above items are complete, this issue may be `/close`'d
 
 Key dates:
-- Initial PR to communtiy repo should be opened by March 1st, {{now.UTC.Year}}
-- PR should be reviewed and merged by April 1st, {{now.UTC.Year}}
+- Initial PR to communtiy repo should be opened by February 14, {{now.UTC.Year}}
+- PR should be reviewed and merged by March 1st, {{now.UTC.Year}}
 
 More detailed information on the annual reports process is available [here](https://git.k8s.io/community/committee-steering/governance/annual-reports.md).
 
+Your group's report template is available here: https://git.k8s.io/community/{{.Dir}}/annual-report-{{lastYear}}.md
```

Comment:
Consider `annual-report-template.md` here?
And then have the generator update the `annual-report-template.md` for the next year.

Reply:
The template will be committed in, so they just PR the changes into that file.